packages/host/app/components/operator-mode/card-inheritance-panel.gts

```diff
@@ -0,0 +1,135 @@
+import Component from '@glimmer/component';
+import { service } from '@ember/service';
+import CardService from '@cardstack/host/services/card-service';
+import {
+  type CardDef,
+  type BaseDef,
+} from 'https://cardstack.com/base/card-api';
+import {
+  type RealmInfo,
+  cardTypeDisplayName,
+  identifyCard,
+  moduleFrom,
+} from '@cardstack/runtime-common';
+import DefinitionContainer, { DefinitionVariant } from './definition-container';
+import { isReady, FileResource } from '@cardstack/host/resources/file';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { ModuleSyntax } from '@cardstack/runtime-common/module-syntax';
+import LoaderService from '@cardstack/host/services/loader-service';
+import moment from 'moment';
+import { type ImportResource } from '@cardstack/host/resources/import';
+
+interface Args {
+  Element: HTMLElement;
+  Args: {
+    realmInfo: RealmInfo | null;
+    realmIconURL: string | null | undefined;
+    openFile: { current: FileResource | undefined };
+    cardInstance: CardDef | null;
+    importedModule?: ImportResource;
+    onCreate?: () => void;
+    onInherit?: () => void;
+    onDuplicate?: () => void;
+  };
+}
+
+export default class CardInheritancePanel extends Component<Args> {
+  @service declare cardService: CardService;
+  @service declare loaderService: LoaderService;
+  @tracked cardInstance: CardDef | undefined;
+  @tracked module: ModuleSyntax | undefined;
+
+  @action
+  duplicateAction() {
+    console.log('running duplicate');
+  }
+
+  @action
+  createAction() {
+    console.log('running create');
+  }
+
+  @action
+  inheritAction() {
+    console.log('running inherit');
+  }
+
+  <template>
+    <div class='container' ...attributes>
+      {{#if @importedModule.module}}
+        {{#each (cardsFromModule @importedModule.module) as |card|}}
+          <DefinitionContainer
+            @name={{this.getCardTypeDisplayName card}}
+            @fileExtension='.GTS'
+            @realmInfo={{@realmInfo}}
+            @realmIconURL={{@realmIconURL}}
+            @variant={{DefinitionVariant.Module}}
+            @onDuplicate={{this.duplicateAction}}
+            @onCreate={{this.createAction}}
+            @onInherit={{this.inheritAction}}
+            @isActive={{false}}
+            data-test-card-module-definition
+          />
+        {{/each}}
+      {{/if}}
+      {{#if @cardInstance}}
+        <DefinitionContainer
+          @name={{@cardInstance.title}}
+          @fileExtension='.JSON'
```

Comment:
i feel ok with this being hardcoded since card instances can only ever be .json files